org.ardour.Ardour5.json

```diff
@@ -0,0 +1,452 @@
+{
+  "app-id": "org.ardour.Ardour5",
+  "runtime": "org.gnome.Platform",
```

Comment:
I was working on a flatpak too and built it against freedesktop runtime successfully.